src/pages/Provider/Provider.scss

```diff
@@ -1,111 +1,127 @@
 .providerHeader {
-  position: relative;
+  position: sticky;
+  top: 0px;
+  z-index: 100;
+  background-color: $secondary;
+  padding: 1.5625rem 2.3125rem 1.125rem;
+  border-radius: 0px 0px 5px 5px;
 
   &__img {
     max-height: 30vh;
-    width: 100%;
-    object-fit: cover;
-    object-position: 50% 50%;
+    width: 4.375rem;
+    height: 1.875rem;
```

Comment:
I think next line don't have correct indentation, please check

Reply:
mmm yes, I think it's ok

![image](https://user-images.githubusercontent.com/66505715/171783351-e6c0ecea-1c0e-4f7f-91ea-d6aa51262f78.png)

Let me know if I'm missing something